Source/OCMockTests/OCMockObjectClassMethodMockingTests.m

```diff
@@ -261,28 +261,28 @@ - (void)testStopMockingDisposesMetaClass
 {
     id mock = [[OCClassMockObject alloc] initWithClass:[TestClassWithClassMethods class]];
 
-    const char *createdSubclassName = object_getClassName([TestClassWithClassMethods class]);
-    XCTAssertNotNil(objc_lookUpClass(createdSubclassName));
+    NSString *createdSubclassName = @(object_getClassName([TestClassWithClassMethods class]));
```

Comment:
I realise that this is one of the changes described in #362. Personally, I feel that converting the C String to an NSString and back purely for the side-effect of copying it, obscures the intent. I'd much prefer to copy the C String, even if it needs a little more code.

Reply:
I can remove this one if you want and save it for when I fully figure out all of the issues with #362. I used the autoboxing to avoid needing to use the C API since that requires string length.

Reply:
I saw what you did ;-) and I understand why it was tempting to use the conversion for its side-effect. I've now pushed a change that stays with C strings.